packages/pangea-sdk/tests/integration/test_authn.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2022 Pangea Cyber Corporation
+# Author: Pangea Cyber Corporation
+
+import random
+import unittest
+
+import pangea.exceptions as pexc
+from pangea import PangeaConfig
+from pangea.services.authn.authn import AuthN
+from pangea.services.authn.models import IDProvider
+from pangea.tools_util import TestEnvironment, get_test_domain, get_test_token
+
+TEST_ENVIRONMENT = TestEnvironment.DEVELOP
+
+
+RANDOM_VALUE = random.randint(0, 10000000)
```

Comment:
we use CI_JOB_ID as the random value in pangea monorepo integration tests, might be worth to standardize?

Reply:
That could a good idea. How should I use that inside a py file?

Reply:
Using CI_JOB_ID would only work in the pipeline, it wouldn't work when running tests locally.

Reply:
Right. I'll leave it as it's